packages/create-sitecore-jss/src/templates/vue/data/routes/styleguide/da-DK.yml

```diff
@@ -0,0 +1,26 @@
+fields:
```

Comment:
See other initializers with language for this file as well. It should be generic, not da-DK specific.